### PR TITLE
Enable FDW, and fix file_fdw tests

### PR DESCRIPTION
**Pull request by adam8157**

### CODE REVIEW

File: src/backend/optimizer/util/pathnode.c [hunk 1]
```diff
@@ -2865,6 +2866,7 @@ create_foreignscan_path(PlannerInfo *root, RelOptInfo *rel,
 	pathnode->path.startup_cost = startup_cost;
 	pathnode->path.total_cost = total_cost;
 	pathnode->path.pathkeys = pathkeys;
+	pathnode->path.locus = cdbpathlocus_from_baserel(root, rel);
```
Comment (macroyuyang) [inline src/backend/optimizer/util/pathnode.c:2869]: For foreign table,  is the locus set correctly from cdbpathlocus_from_baserel?

Comment (adam8157) [reply src/backend/optimizer/util/pathnode.c:2869]: I think it's correct, just like external table, locus is Entry.

**Comment by paul-guo-** [comment]
Seems good.

**Comment by paul-guo-** [comment]
Do we need to add some MPP related test cases?

**Comment by adam8157** [comment]
FDW has no MPP capability yet I suppose.

**Review by macroyuyang — APPROVED** [review]

**Pull request merged**